TheInternet.Tests/HoverTests.cs

```diff
@@ -16,5 +16,13 @@ public void Hovering_over_each_thingy()
         Assert.AreEqual("name: user1", hoversPage.hoverOveruserImage(1).getUsername());
         Assert.AreEqual("https://the-internet.herokuapp.com/users/1", hoversPage.hoverOveruserImage(1).getUserLink());
 
+        Assert.True(hoversPage.hoverOveruserImage(2).isCaptionDisplayed());
```

Comment:
This might be an opportunity to play with variables and a for loop - this appears to be the same code repeated 3 times with an incrementing user number. If you decide to test another 2 rows it then becomes a simple matter of changing the increment count rather that copy-paste-change-change-change.

If you have a go at that, remember you can interpolate the variable into the string like so: 

```
Assert.AreEqual($"https://the-internet.herokuapp.com/users/{userNumber}", hoversPage.hoverOveruserImage(userNumber).getUserLink());
```

(note the $ in front of the string)